Zend/zend.c

```diff
@@ -1765,3 +1767,41 @@ ZEND_API void zend_map_ptr_extend(size_t last)
 		CG(map_ptr_last) = last;
 	}
 }
+
+static void zend_error_notify_callback_dtor(zend_error_notify_callback *callback)
+{
+}
+
+int zend_startup_error_notify_callbacks()
+{
+	zend_llist_init(&zend_error_notify_callbacks, sizeof(zend_error_notify_callback), (void (*)(void *)) zend_error_notify_callback_dtor, 1);
+
+	return SUCCESS;
+}
+
+int zend_shutdown_error_notify_callbacks()
+{
+	zend_llist_destroy(&zend_error_notify_callbacks);
+
+	return SUCCESS;
+}
```

Comment:
```suggestion
void zend_shutdown_error_notify_callbacks()
{
	zend_llist_destroy(&zend_error_notify_callbacks);
}
```
Wouldn't this be better as it seems that it can't fail? Same for the startup function.